libbeat/common/config.go

```diff
@@ -327,33 +328,49 @@ func (f *flagOverwrite) Get() interface{} {
 	return f.value
 }
 
-// Validate checks at most one sub-namespace being set.
-func (ns *ConfigNamespace) Validate() error {
-	if len(ns.C) > 1 {
-		return errors.New("more then one namespace configured")
+func (ns *ConfigNamespace) Unpack(cfg *Config) error {
```

Comment:
Could you add here some docs on what is happening if more then 1 namespace is enabled?